wiki/History_of_osu!/2011/en.md

```diff
@@ -1,9 +1,54 @@
-# History of osu! 2011
+---
+stub: true
+---
+
+# 2011
+
+## February
+
+New features were added to the osu! website, including beatmap listing filters for played beatmaps ([osu!supporter](/wiki/osu!supporter) only), beatmap packs were sorted into categories, and avatars became visible in the header.[^web-update]
```

Comment:
What does `header` mean here? User panel preview?

Reply:
this would be the header. it might've displayed the avatar when logged in but i'm not sure

![image](https://github.com/ppy/osu-wiki/assets/36758269/5d632a99-06b9-4058-80d6-4b61fc32ebe2)

![image](https://github.com/ppy/osu-wiki/assets/36758269/6016b580-d25e-480c-9fbf-5e3652eb56cb)

Reply:
it did, yep.